docs/electronics/ultimachine.md

```diff
@@ -114,8 +114,6 @@ Need a fancy Box for your fancy board?
 
 [Gnarly Mini-Rambo Case](https://www.thingiverse.com/thing:2778151)
 
-![!case](https://www.thingiverse.com/thing:2778151)
```

Comment:
This should probably be changed to an image of the mini case

Reply:
Hmmm where did the picture go?